go/host/db/hostdb.go

```diff
@@ -285,20 +285,20 @@ func (db *DB) writeHeadRollup(w ethdb.KeyValueWriter, val gethcommon.Hash) {
 	}
 }
 
-// Stores the hash of a rollup into the database, keyed by the rollup's number
+// Stores a rollup's hash in the database, keyed by the rollup's number.
 func (db *DB) writeRollupHash(w ethdb.KeyValueWriter, header *common.Header) {
 	key := rollupHashKey(header.Number)
 	if err := w.Put(key, header.Hash().Bytes()); err != nil {
 		db.logger.Crit("could not put header in DB.", log.ErrKey, err)
 	}
 }
 
-// Stores the hash of a rollup into the database, keyed by the hashes of the transactions in the rollup
-func (db *DB) writeRollupNumber(w ethdb.KeyValueWriter, txHash gethcommon.Hash, rollupNumber *big.Int) {
+// Stores a rollup's number in the database, keyed by the hash of a transaction in that rollup.
```

Comment:
This description change is the gist of the PR. Was confusing me for a bit what was meant here.